src/utils/CaminoExporter.ts

```diff
@@ -81,7 +81,6 @@ export class CaminoExporter {
                     cql: libCql.cql,
                     version: libCql?.version || ''
                   };
-                  referencedDefines[libCql.cql] = libName;
```

Comment:
Not sure I understand why this is being removed

Reply:
This is the fix for # 3 in my list above. `libCql.cql` is the entire content of the referenced cql library. And then look at how `referencedDefines` is used in the `constructCqlLibrary` function:

```
  const definesList = Object.entries(referencedDefines).map(
    ([name, srcLibrary]) => `define "${name}": ${srcLibrary}."${name}"\n\n`
  );
```
This is meant to just re-map definitions from a dependency library, so like `define "T > 2": TNM."T>2"`

And I'm pretty sure everything we need is included elsewhere, so this line can just be removed.
The relevant bits to make that remapping work for this specific criteria are set a few lines above:
```
referencedDefines[transition.condition.cql] = libraryIdentifier.id;
```
and the relevant library gets included by this part:
```
includedCqlLibraries[libName] = {
  cql: libCql.cql,
  version: libCql?.version || ''
};
```